ext/ITensorNetworksEinExprsExt/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,50 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, @Algorithm_str, noncommoninds
```

Comment:
```suggestion
using ITensors: Index, ITensor, @Algorithm_str, inds, noncommoninds
```
Fix another namespace issue.